en/index.md

```diff
@@ -1,33 +1,15 @@
 ---
 title: Elixir School
```

Comment:
Should Elixir be changed to Haskell here?

Reply:
Curse, yes